<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogConfig</class>
 <widget class="QDialog" name="DialogConfig">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>463</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qtradingview/icons/iconos.qrc">
    <normaloff>:/actions/settings</normaloff>:/actions/settings</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <property name="text">
        <string>Language</string>
       </property>
       <property name="margin">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_languages">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Initial market</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_initial_market">
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_initial_exchange">
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="list_exchanges">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>150</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectItems</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="movement">
      <enum>QListView::Static</enum>
     </property>
     <property name="flow">
      <enum>QListView::TopToBottom</enum>
     </property>
     <property name="spacing">
      <number>3</number>
     </property>
     <property name="viewMode">
      <enum>QListView::ListMode</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>Binance</string>
      </property>
      <property name="icon">
       <iconset resource="../qtradingview/icons/iconos.qrc">
        <normaloff>:/exchanges/binance</normaloff>:/exchanges/binance</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>BitBay</string>
      </property>
      <property name="icon">
       <iconset resource="../qtradingview/icons/iconos.qrc">
        <normaloff>:/exchanges/bitbay</normaloff>:/exchanges/bitbay</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bitfinex</string>
      </property>
      <property name="icon">
       <iconset resource="../qtradingview/icons/iconos.qrc">
        <normaloff>:/exchanges/bitfinex</normaloff>:/exchanges/bitfinex</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bithumb</string>
      </property>
      <property name="icon">
       <iconset resource="../qtradingview/icons/iconos.qrc">
        <normaloff>:/exchanges/bithumb</normaloff>:/exchanges/bithumb</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bitso</string>
      </property>
      <property name="icon">
       <iconset resource="../qtradingview/icons/iconos.qrc">
        <normaloff>:/exchanges/bitso</normaloff>:/exchanges/bitso</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bitstamp</string>
      </property>
      <property name="icon">
       <iconset resource="../qtradingview/icons/iconos.qrc">
        <normaloff>:/exchanges/bitstamp</normaloff>:/exchanges/bitstamp</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bittrex</string>
      </property>
      <property name="icon">
       <iconset resource="../qtradingview/icons/iconos.qrc">
        <normaloff>:/exchanges/bittrex</normaloff>:/exchanges/bittrex</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Coinbase</string>
      </property>
      <property name="icon">
       <iconset resource="../qtradingview/icons/iconos.qrc">
        <normaloff>:/exchanges/coinbase</normaloff>:/exchanges/coinbase</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Gemini</string>
      </property>
      <property name="icon">
       <iconset resource="../qtradingview/icons/iconos.qrc">
        <normaloff>:/exchanges/gemini</normaloff>:/exchanges/gemini</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>HitBTC</string>
      </property>
      <property name="icon">
       <iconset resource="../qtradingview/icons/iconos.qrc">
        <normaloff>:/exchanges/hitbtc</normaloff>:/exchanges/hitbtc</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Kraken</string>
      </property>
      <property name="icon">
       <iconset resource="../qtradingview/icons/iconos.qrc">
        <normaloff>:/exchanges/kraken</normaloff>:/exchanges/kraken</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Kucoin</string>
      </property>
      <property name="icon">
       <iconset resource="../qtradingview/icons/iconos.qrc">
        <normaloff>:/exchanges/kucoin</normaloff>:/exchanges/kucoin</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>OKCoin</string>
      </property>
      <property name="icon">
       <iconset resource="../qtradingview/icons/iconos.qrc">
        <normaloff>:/exchanges/okcoin</normaloff>:/exchanges/okcoin</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Poloniex</string>
      </property>
      <property name="icon">
       <iconset resource="../qtradingview/icons/iconos.qrc">
        <normaloff>:/exchanges/poloniex</normaloff>:/exchanges/poloniex</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TheRock</string>
      </property>
      <property name="icon">
       <iconset resource="../qtradingview/icons/iconos.qrc">
        <normaloff>:/exchanges/therock</normaloff>:/exchanges/therock</iconset>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogConfig</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogConfig</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
